storage/cloud.py

```diff
@@ -696,6 +696,28 @@ def cancel_chunked_upload(self, uuid, storage_metadata):
         for chunk in self._chunk_list_from_metadata(storage_metadata):
             self.remove(chunk.path)
 
+    def clean_partial_uploads(self, deletion_date_threshold):
```

Comment:
I still think that this is something that should be handled by the storage service itself, and not Quay, since we don't really track partial uploaded blob storage, and we're essentially reimplementing a feature that's already available.

Reply:
Or at least not on by default.

Reply:
Yeah I agree it shouldn't be enabled by default. I can also see the usefulness in not having the operational overhead of setting up retention rules, something that works out of the box. We can discuss more with the rest of the team if you want.